metricbeat/module/jolokia/jmx/_meta/data.json

```diff
@@ -4,30 +4,34 @@
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "jolokia.jmx",
+        "duration": 115000
+    },
     "jolokia": {
         "testnamespace": {
             "memory": {
                 "heap_usage": {
-                    "committed": 112721920,
-                    "init": 64673792,
-                    "max": 921174016,
-                    "used": 81023984
+                    "committed": 514850816,
+                    "init": 536870912,
+                    "max": 7635730432,
+                    "used": 48847152
                 },
                 "non_heap_usage": {
-                    "committed": 24576000,
-                    "init": 24576000,
-                    "max": 224395264,
-                    "used": 17684240
+                    "committed": 29949952,
+                    "init": 2555904,
+                    "max": -1,
+                    "used": 28903168
                 }
             },
-            "uptime": 580487
+            "uptime": 1538717
         }
     },
     "metricset": {
-        "host": "jolokia:8778",
+        "host": "127.0.0.1:8080",
         "module": "jolokia",
         "name": "jmx",
-        "namespace": "testnamespace",
```

Comment:
Umm, this is changing :thinking: 

Though I don't think this is related with this change.

Reply:
Yeah I regenerated `data.json` without this backport pr and namespace got changed as well hmm

Reply:
This is a change we did quite some time ago. Seems like we never regenerated the file since then 🤔

Reply:
good to know! thanks!! Next time we will make sure to regenerate `data.json` 😄